### DISCUSSION TITLE
Keymap for pubg mobile lastest version

### DISCUSSION THREAD

**Discussion by DarqPL** [barry-ran/QtScrcpy#992] [Q&A]
Anyone have keymap file for pubg mobile lastest version please, the keymap for gameofpeace can't use for pubg mobile. (Sorry for bad english)

**Comment by hsain9357** [comment]
you can take a screenshot of your control and head over to [this website](https://qtscrcpy-keymap.netlify.app/) upload your pic and make your own key mapping

**Comment by cawhyred** [comment]
@hsain9357 the link for the website is down

**Comment by CodeXTF2** [comment]
made a crappy tool to do the same locally:

https://github.com/CodeXTF2/qtscrcpy_keymap_editor

**Comment by w4po** [comment]
Hi, Sorry for the late response but I've just made a visual key mapper for QtScrcpy: https://github.com/w4po/ScrcpyKeyMapper
You can now create your own <3
